code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -853,7 +853,11 @@ var/list/ai_verbs_default = list(
 				if("ancient machine")
 					holo_icon = getHologramIcon(icon('icons/mob/ancient_machine.dmi', "ancient_machine"))
 				if("custom")
-					holo_icon = getHologramIcon(icon('icons/mob/custom-synthetic.dmi', "[ckey]-ai-holo"))
+					if("[ckey]-ai-holo" in icon_states('icons/mob/custom-synthetic.dmi'))
+						holo_icon = getHologramIcon(icon('icons/mob/custom-synthetic.dmi', "[ckey]-ai-holo"))
+					else

```

Comment:
Any reason why there's no validation to check if("[ckey]-ai-holo" in icon_states('icons/mob/custom-synthetic64.dmi'))?